test/e2e/install_test.go

```diff
@@ -3,20 +3,37 @@ package e2e
 import (
 	"context"
 	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	operatorv1alpha1 "github.com/operator-framework/operator-controller/api/v1alpha1"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-
+	"gopkg.in/yaml.v2"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	apimeta "k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/rand"
+	kubeclient "k8s.io/client-go/kubernetes"
+	"k8s.io/utils/env"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	catalogd "github.com/operator-framework/catalogd/api/core/v1alpha1"
 	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+)
 
-	operatorv1alpha1 "github.com/operator-framework/operator-controller/api/v1alpha1"
+const (
+	defaultTimeout = 30 * time.Second
+	defaultPoll    = 1 * time.Second
+	testCatalogRef = "localhost/testdata/catalogs/test-catalog:e2e"
```

Comment:
This needs to be removed - e2e_suite_test.go currently has consts for this, and a helper function `getCatalogImageRef` which supports optionally changing the catalog image via an env var.

Reply:
done